sherpa/utils/src/_psf.cc

```diff
@@ -273,48 +294,30 @@ static PyObject* tcdPyData_convolve( tcdPyData* self, PyObject* args )
 			  &center) )
     return NULL;
   
-  if( dims_src.get_size() != dims_kern.get_size() ) {
-    std::ostringstream err;
-    err << "input array sizes do not match, "
-	<< "dims_src: " << dims_src.get_size()
-	<< " vs dims_kern: " << dims_kern.get_size();
-    PyErr_SetString( PyExc_TypeError, err.str().c_str() );
+  if( !same_size_arrays( dims_src.get_size(), dims_kern.get_size(),
+                         ", ", "dims_src", " vs dims_kern" ) )
```

Comment:
the last parameter in the call to same_size_arrays(),  " vs dims_kern" should only be "dims_kern". The " vs " portion is formatting that should be handled in same_size_arrays. This comment applies to all 13 instances of calls to same_size_arrays in this PR.